projects/msvc16/unittest/stdafx.cpp

```diff
@@ -0,0 +1,8 @@
+// stdafx.cpp : source file that includes just the standard includes
+// iutest_unittest.pch will be the pre-compiled header
+// stdafx.obj will contain the pre-compiled type information
+
+#include "stdafx.h"
+
+// TODO: reference any additional headers you need in STDAFX.H
```

Comment:
__[Goodcheck]__ Complete all work!
<details>


Based on the comments it looks like some work was left for later.
Are you sure you want to continue?

</details>
<sub>posted by Sider</sub>